app/accounts/views.py

```
@@ -139,10 +139,15 @@ def form_invalid(self, form):
         return super().form_invalid(form)
 
 
-class LogoutViewCBV(LogoutView):
-    """Vue de déconnexion simple."""
+from django.contrib.auth import logout
+from django.contrib import messages
```

Ruff (0.12.2)

Redefinition of unused `messages` from line 4

Remove definition: `messages`

(F811)